README.md

```diff
@@ -1,81 +1,173 @@
 
-1. Clone this repository and navigate to the directory:
+Clone this repository and navigate to the directory:
 ```
 git clone git@github.com:Sage-Bionetworks/genie-bpc-pipeline.git
-cd genie-bpc-pipeline
+cd genie-bpc-pipeline/scripts/case_selection
 ```
 
-2. Install Nextflow and Docker.  Instructions are available at the following links: 
-
-- https://www.nextflow.io/docs/latest/getstarted.html
-- https://docs.docker.com/get-docker/
+Install all required R packages:
+```
+R -e 'renv::restore()'
+```
 
 ## Synapse credentials
 
-Cache your Synapse personal access token (PAT) as a secret named `SYNAPSE_AUTH_TOKEN` or store in a file named `.synapseConfig` with the following format:
+Cache your Synapse personal access token (PAT) as an environmental variable:
+```
+export SYNAPSE_AUTH_TOKEN={your_personal_access_token_here}
+```
+
+or store in ~/.synapseConfig with the following format:
 ```
 [authentication]
 
-# cache authtoken below
+# either authtoken OR username and password
 authtoken = {your_personal_access_token_here}
 ```
 
-## Usage
+## Usage: case selection
+
+To display the command line interface:
+```
+Rscript workflow_case_selection.R -h
+```
+
+The command line interface will display as follows:
+```
+Usage: workflow_case_selection.R [options]
+
+
+Options:
+	-p PHASE, --phase=PHASE
+		BPC phase
+
+	-c COHORT, --cohort=COHORT
+		BPC cohort
+
+	-s SITE, --site=SITE
+		BPC site
+
+	-u, --save_synapse
+		Save output to Synapse
+
+	-h, --help
+		Show this help message and exit
+```
+
+Example run: 
+```
+Rscript workflow_case_selection.R  -p 1 -c NSCLC -s DFCI -u
+```
+
+## Usage: create GENIE export files
+
+To display the command line interface:
+```
+Rscript export_bpc_selected_cases.R -h
+```
+
+The command line interface will display as follows:
+```
+Usage: export_bpc_selected_cases.R [options]
+
+
+Options:
+	-i INPUT, --input=INPUT
+		Synapse ID of the input file that has the BPC selected cases
+
+	-o OUTPUT, --output=OUTPUT
+		Synapse ID of the BPC output folder. Default: syn20798271
 
-To run the Nextflow pipeline:
+	-p PHASE, --phase=PHASE
+		BPC phase. i.e. pilot, phase 1, phase 1 additional
+
+	-c COHORT, --cohort=COHORT
+		BPC cohort. i.e. NSCLC, CRC, BrCa, and etc.
+
+	-s SITE, --site=SITE
+		BPC site. i.e. DFCI, MSK, UHN, VICC, and etc.
+
+	-h, --help
+		Show this help message and exit
 ```
-nextflow run main.nf --cohort {cohort} --comment {comment}
+
+## Usage: update table counts
+
+To display the command line interface:
+```
+Rscript update_case_count_table.R -h
 ```
 
-Input parameters:
-- `cohort`: BPC cohort code (for valid cohort codes see `nextflow.config`)
-- `comment`: message to use for Synapse table snapshots regarding the update (e.g. "NSCLC public release")
-- `synapseConfig`: (optional) path to `.synapseConfig` file.  If not specified, program uses `SYNAPSE_AUTH_TOKEN` secret.
+The command line interface will display as follows:
+```
+Usage: update_case_count_table.R [options]
 
-## DockerHub builds
```

Comment:
@Chelsea-Na can we make sure we add this back in, are the case selections a nextflow workflow that can be executed?